src/app/accounts/account-limit.ts

```diff
@@ -0,0 +1,81 @@
+import { volumesForAccountId } from "@app/payments/helpers"
+import { getMidPriceRatio } from "@app/shared"
+import { getAccountLimits, getDealerConfig, ONE_DAY } from "@config"
+import { AccountLimitsVolumes } from "@domain/accounts/limits-volume"
+import { InvalidAccountLimitTypeError } from "@domain/errors"
+import { LedgerService } from "@services/ledger"
+
+export const accountLimit = async ({
+  account,
+  limitType,
+}: {
+  account: Account
+  limitType: "Intraledger" | "Withdrawal" | "TradeIntraAccount"
```

Comment:
looks like `LimitType` should be a galoy style enum with hard coded values.
```
let const LimitType = {
  IntraLedger: "IntraLedger",
  ...
} as const
```

don't know if ^^ that is the exact syntax we've been using - but I'm sure you know what I'm referring to. Then we derive a type and reference `LimitType.IntraLedger` instead of hard coded strings everywhere.

Reply:
https://github.com/GaloyMoney/galoy/pull/1953/commits/e82fb51f3f67e7cef82fe468968876c57d16c8d8